src/main/resources/config.patch.yml

```diff
@@ -3,3 +3,7 @@
   - type: SET
     key: journal.custom_model_data
     value: -1
+  - type: SET
```

Comment:
This must be in it's own config version so that everyone that is already on 2.0.0-CONFIG-1 also get's these changes.